GameServerApp/Settings/GameInfo.json

```diff
@@ -0,0 +1,58 @@
+{
```

Comment:
Should this be here?

If we are still using `.template` approach (I have been afk for a long time, idk), this shouldn't be here. Please modify `.gitignore` accordingly.

Reply:
Fixed, and updated .gitignore to ignore this Settings folder (maybe should ignore all GameApp).